src/IW/Db/Repo.hs

```diff
@@ -0,0 +1,32 @@
+{-# LANGUAGE QuasiQuotes #-}
+
+-- | SQL queries to work with the @issues@ table.
```

Comment:
I guess it was just copy-paste. I wish we had something like GHC compiler for the documentation as well...
```suggestion
-- | SQL queries to work with the @repos@ table.
```